rosbridge_library/src/rosbridge_library/capabilities/fragmentation.py

```diff
@@ -77,7 +77,7 @@ def fragment(self, message, fragment_size, mid=None):
         if message_length <= fragment_size:
             return [message]
 
-        msg_id = message.get("id", None)
```

Comment:
I suggest just deleting this line entirely

Reply:
DOne.